### PR TITLE
Formalize holidays and time off policy

### PR DESCRIPTION
**Pull request by dcwalk**
This was in our business plan, and this is the time off policy we've been working with around office closures and notification. 

Prompted because we almost missed another holiday again? This shouldn't be merged until discussed at all hands, but codifies existing practices and comments from our business strategy. 

Once approved I will update our calendar and merge

### CODE REVIEW

**Comment by dcwalk** [comment]
Prompted because we missed another holiday again? This shouldn't be merged until discussed at all hands, but codifies existing practices and comments from our business strategy

**Review by patcon — APPROVED** [review]
Reviewed and LGTM! Thanks for much, @dcwalk :)

File: holidays.md
Comment (patcon) [inline holidays.md:6]:
```suggestion
and member-workers are entitled (encouraged) to those days off and receive an [annualized public holiday pay](finance.md#public-holiday-pay). 
```

**Comment by patcon** [comment]
Some but not all canada.ca links are failing. As per https://github.com/gjtorikian/html-proofer/issues/131#issue-48437225, how about we dial down concurrency here, and see if that reduces these annoying errors errors:

https://github.com/hyphacoop/handbook/blob/45d354ec90691ad913793706b4a03b708174e44f/Makefile#L6-L10

**Comment by patcon** [comment]
Hm. Still not working. Will try to get concurrency working when i get back on compy in 30 min, or otherwise will revert back to unforked html-proofer (you're welcome to do same)

**Comment by patcon** [comment]
ok fixed :)

File: README.md
Comment (patcon) [inline README.md:53]:
```suggestion
To test that all links works, you may run html-proofer via:
```

**Comment by dcwalk** [comment]
Just the list of holidays to work from for calendar:
Holiday 	2020 	2021 	202
New Year's Day
January 1 	
Wed, January 1 	
Fri, January 1 	
Sat, January 1

Ontario Family Day
Third Monday in February 	
Mon, February 17 	
Mon, February 15 	
Mon, February 21

Good Friday
Friday before Easter Sunday 	
Fri, April 10 	
Fri, April 2 	
Fri, April 15

May Day (International Workers' Day)
May 1 	
Friday, May 1 	
Saturday, May 1 	
Sunday, May 1

Victoria Day
Monday before May 25 	
Mon, May 18 	
Mon, May 24 	
Mon, May 23

Canada Day
July 1 	
Wed, July 1 	
Thu, July 1 	
Fri, July 1

Civic Holiday
First Monday in August 	
Mon, August 3 	
Mon, August 2 	
Mon, August 1

Labour Day
First Monday in September 	
Mon, September 7 	
Mon, September 6 	
Mon, September 5

Thanksgiving
Second Monday in October 	
Mon, October 12 	
Mon, October 11 	
Mon, October 10

Remembrance Day
November 11 	
Wed, November 11 	
Thu, November 11 	
Fri, November 11

Christmas Day
December 25 	
Fri, December 25 	
Sat, December 25 	
Sun, December 25

Boxing Day
December 26 	
Sat, December 26 	
Sun, December 26 	
Mon, December 26

**Review by benhylau — APPROVED** [review]

**Comment by dcwalk** [comment]
We should consider Earth Day apparently (April 22).

**Comment by dcwalk** [comment]
Consensed / confirmed this was our policy at 2021-04-07 all hands

File: Gemfile
Comment (dcwalk) [inline Gemfile:6]: I feel a little cautious about this approach

Comment (patcon) [reply Gemfile:6]: ok, def remove it then. i've shared my logic, and your call 🤷

Comment (patcon) [reply Gemfile:6]: wait, no, i thought this was something else.

Sorry, why feeling cautious? This is the self-documenting way I understand folks use package managers when waiting for upstream changes to get merged, and i'm not sure how else one goes about it... waiting on upstream to merge months later is not an approach I'm familiar with. Would forking to hyphacoop org feel more solid?

Comment (patcon) [reply Gemfile:6]: If you want to hold off a bit longer, it will prob be merged in the next week: https://github.com/gjtorikian/html-proofer/pull/632#issuecomment-810670733

But merging stuff like this from in-process forks is totally par for the course in past positions I've worked at, and isn't usually considered a "code smell". It's just house-keeping to go through the lockfile every few months or so and sweep back to mainline or poke PR's along :)

**Pull request merged**
